controller/user.js

```diff
@@ -81,14 +91,42 @@ const create = async (req, res, next) => {
 
     const hash = await hashPassword(password);
 
-    const newUser = { ...newUserTemplate, password: hash };
+    const verificationEmailToken = uuidv4();
+
+    const newUser = {
+      ...newUserTemplate,
+      password: hash,
+      verificationToken: verificationEmailToken,
+    };
 
     await service.addUser(newUser);
 
+    // await service.storeVerificationEmailToken(verificationEmailToken);
+
+    const emailMessage = {
+      to: "apolinary@poczta.onet.pl",
+      from: "apolinary@poczta.onet.pl",
+      subject: "Check your email to complete verification",
+      text: `Your link to verificate email: ${verificationEmailToken}`,
+      html: `<h1>Your link to verificate email: ${verificationEmailToken}</h1>`,
```

Comment:
```suggestion
      html: `<h1>Your link to verificate email: ${verificationEmailToken}</h1>`,
```
Tu jakiś ładny html na przyszłość :) tak aby był button albo <a> tag.

Reply:
+

Reply:
Starałem się ogarnąć wszystkie rzeczy wymienione w filmiku, testowałem postmanem zobaczymy czy wystarczająco dobrze :))